app/graphql/types/query_types/point_of_interest_type.rb

```diff
@@ -24,9 +24,27 @@ class QueryTypes::PointOfInterestType < Types::BaseObject
     field :certificates, [QueryTypes::CertificateType], null: true
     field :accessibility_information, QueryTypes::AccessibilityInformationType, null: true
     field :tag_list, [String], null: true
+    field :has_travel_times, Boolean, null: true do
+      argument :date, String, required: false
+    end
+    field :travel_times, [QueryTypes::PublicTransportation::TravelTimesType], null: true do
+      argument :date, String, required: false
+    end
     field :lunches, [QueryTypes::LunchType], null: true
     field :payload, GraphQL::Types::JSON, null: true
     field :updated_at, String, null: true
     field :created_at, String, null: true
+
+    def has_travel_times(date: "")
+      return false if date.blank?
+
+      object.gtfs_travel_times(date: date).present?
+    end
```

Comment:
we need to think about something easier for returning true or false here, instead of requesting alle the data to know that.